### ISSUE TITLE
Piece-Square Tables evaluation

### ISSUE THREAD

**Issue by Varsius** [healeycodes/andoma#12]
When you take a new board and for example just play the move "e2e4" and afterwards call evaluate_board, it returns a value of -25, which means black is slightly favoured. That would not make sense, because white just moved a pawn to a better square and nothing else happend.

Code used:
```import chess
from evaluate import evaluate_board
board = chess.Board()
board.push(chess.Move.from_uci("e2e4"))
evaluate_board(board)
# output: -25
```

After a bit of digging I noticed that the for-loop in evaluate_board
`for square in chess.SQUARES` is equivalent to `for square in range(64)`.
This means the first square is 0 and therefore would be the first entry in one of the piece-square-tables for example `pawnEvalWhite` which is square a8.
But if you call `board.piece_at(0)` you get the piece at square a1 instead.

Am I missing something or should the piece-square tables be reversed?

**Comment by Varsius** [comment]
Here is the documentation entry I found: https://python-chess.readthedocs.io/en/latest/core.html#square-sets

But I think you can not just simply reverse your tables, you have to reverse the rows instead. For example:

```

# Old piece table
pawnEvalWhite = [
    0, 0, 0, 0, 0, 0, 0, 0,
    50, 50, 50, 50, 50, 50, 50, 50,
    10, 10, 20, 30, 30, 20, 10, 10,
    5, 5, 10, 25, 25, 10, 5, 5,
    0, 0, 0, 20, 20, 0, 0, 0,
    5, -5, -10, 0, 0, -10, -5, 5,
    5, 10, 10, -20, -20, 10, 10, 5,
    0, 0, 0, 0, 0, 0, 0, 0,
]

# New piece table with reversed rows
pawnEvalWhite = [    
    0,  0,  0,  0,  0,  0,  0,  0,
    5, 10, 10, -20, -20, 10, 10,  5,
    5, -5, -10,  0,  0, -10, -5,  5,
    0,  0,  0, 20, 20,  0,  0,  0,
    5,  5, 10, 25, 25, 10,  5,  5,
    10, 10, 20, 30, 30, 20, 10, 10,
    50, 50, 50, 50, 50, 50, 50, 50,
    0, 0, 0, 0, 0, 0, 0, 0
]
```

**Comment by healeycodes** [comment]
Hi @Varsius!

Thanks for raising this and for the detailed write up. It looks like you're onto something.

If you have a fix in mind – would you like to raise a PR? If not, I'll pick this up later.

**Comment by Varsius** [comment]
I will create a PR in the next couple days 👍

**Comment by healeycodes** [comment]
Fixed by #13
